.travis.yml

```diff
@@ -1,5 +1,68 @@
-language: node_js
-node_js:
-  - "node"
+sudo: false
+
+language: php
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+branches:
+  only:
+    - master
+
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
+
+env:
+  - TRAVIS_NODE_VERSION="7"
+
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
+
+matrix:
+  include:
+    - php: 7.1
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=latest
+    - php: 5.2
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: TRAVISCI=phpcs
+    - php: 5.6
```

Comment:
Let's use the latest PHP 7.1 for this (WP Core uses the latest for its JS only job)